paymentsheet/api/paymentsheet.api

```diff
@@ -417,6 +417,12 @@ public final class com/stripe/android/link/ui/ComposableSingletons$PrimaryButton
 	public final fun getLambda-1$paymentsheet_release ()Lkotlin/jvm/functions/Function2;
 }
 
+public final class com/stripe/android/link/ui/ComposableSingletons$SecondaryButtonKt {
```

Comment:
👀 known issue: https://github.com/Kotlin/binary-compatibility-validator/issues/74

Reply:
Yeah, this is expected and accepted by the team! You'll see api changes frequently as you work on Compose in both FC and MPE.

Parcelable changes also get reflected on the api files. That's expected as well!